packages/cli/src/util/output/create-output.ts

```diff
@@ -49,15 +61,19 @@ export class Output {
     str: string,
     slug: string | null = null,
     link: string | null = null,
-    action: string | null = 'Learn More'
+    action: string = 'Learn More'
   ) => {
     const details = slug ? `https://err.sh/vercel/${slug}` : link;
 
     this.print(
       chalk.yellow(
         chalk.bold('WARN! ') +
           str +
-          (details ? `\n${action}: ${renderLink(details)}` : '')
+          (details
+            ? `\n${this.link(action, details, {
+                fallback: () => `${action}: ${renderLink(details)}`,
```

Comment:
How will users know to click the link? I think we need to be careful to only use this when the link matches the text.

Reply:
I reverted the change to `warn()` and `error()`. We can discuss that at a later time.